app/assets/javascripts/app/core/me.js

```diff
@@ -0,0 +1,37 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('council.core')
+    .service('Me', Me);
+
+  function Me($http, $q) {
+    var me;
+    var mePromise;
+
+    function find() {
+      mePromise = $http.get('/me')
+        .then(set);
+      return mePromise;
+    }
+
+    function set(response) {
+      me = response.data;
+      return me;
```

Comment:
> Do you need to return this?

I'm returning `me` here to make it the value of `mePromise` in line 13 (https://docs.angularjs.org/api/ng/service/$q#chaining-promises).